app/Providers/AppServiceProvider.php

```diff
@@ -19,6 +19,8 @@ public function boot()
             'monster' => 'App\Models\Monster',
             'user'    => 'App\User',
         ]);
+
+        // CauserResolver::setCauser(User::first());
```

Comment:
is this needed ? a leftover or an example ?

Reply:
Leftover...
Activity Log package now handles that by it self 👌